tools/test_configs/RealtekInterface.json

```diff
@@ -9,7 +9,7 @@
         },
         "connect-statement" : {
             "help" : "Disabled until EMAC updated",
-            "value" : "((RTWInterface *)net)->connect(WIFI_SSID, WIFI_PASSWORD, WIFI_SECURITY, WIFI_CHANNEL)"
+            "value" : "(net->wifiInterface()->connect(WIFI_SSID, WIFI_PASSWORD, WIFI_SECURITY, WIFI_CHANNEL)"
```

Comment:
Unless I'm mistaken, this has an excess '(' at the front.

Could you put back the original help from master too:

    "help": "Must use 'net' variable name, replace WIFI_SSID, WIFI_PASSWORD, WIFI_SECURITY, WIFI_CHANNEL with your WiFi settings",

Reply:
Ok.